recordset/recordset.html

```diff
@@ -13,6 +13,12 @@
                     uib-tooltip="{{ ((vm.pageLimit > vm.RECORDEDIT_MAX_ROWS) ? 'Editing disabled when items per page > ' + vm.RECORDEDIT_MAX_ROWS : 'Click here to edit this page of records.' ) }}">
                     <span class="glyphicon glyphicon-pencil"></span> <strong>Edit</strong>
                 </a>&nbsp;&nbsp;
+                <!-- Download Link -->
+                <a id="downloadCSV-link" ng-if="vm.rowValues.length>0" ng-href="{{ csvLink() }}">
```

Comment:
Instead of using the function you defined on `$scope`, you could use `vm.reference.csvDownloadLink` in the `ng-href` here.